src/components/PageList.vue

```diff
@@ -174,8 +194,19 @@ export default {
 			'sortBy',
 			'showing',
 			'showTemplates',
+			'allPages',
 		]),
 
+		allpages() {
+			return this.allPages(this.rootPage.collectivePath)
+		},
+
+		filteredPages() {
```

Comment:
Could this just use the store getter?

Reply:
Ah no, I guess it needs to be based on the collective path, but then the store one is unused

Reply:
the unused store getter is removed.